Fail to build Fabric8 :: Java generator :: Gradle Plugin

### Describe the bug

Building the project using `mvn clean package -DskipTests` and fails with:
```
...
17:24:35.433 [INFO] Fabric8 :: Java generator :: Gradle Plugin ......... FAILURE [  0.082 s]
17:24:35.433 [INFO] Fabric8 :: Java generator :: Maven Plugin .......... SKIPPED
17:24:35.433 [INFO] Fabric8 :: Java generator :: Integration Tests ..... SKIPPED
17:24:35.433 [INFO] Fabric8 :: Java generator :: Benchmarks ............ SKIPPED
17:24:35.433 [INFO] Fabric8 :: Kubernetes :: HttpClient :: Vert.x ...... SKIPPED
17:24:35.433 [INFO] Fabric8 :: Kubernetes :: Dependency Compatibility :: Tests SKIPPED
17:24:35.433 [INFO] Fabric8 :: Kubernetes :: Bouncy Castle FIPS Compatibility :: Test SKIPPED
17:24:35.433 [INFO] Fabric8 :: Kubernetes :: Log4j Core components ..... SKIPPED
17:24:35.433 [INFO] Fabric8 :: Kubernetes :: HttpClient :: JDK ......... SKIPPED
17:24:35.433 [INFO] Fabric8 :: Kubernetes :: HttpClient :: Jetty ....... SKIPPED
17:24:35.433 [INFO] Fabric8 :: Kubernetes :: HttpClient Tests .......... SKIPPED
17:24:35.433 [INFO] Kube API Test Parent ............................... SKIPPED
17:24:35.433 [INFO] Kube API Test ...................................... SKIPPED
17:24:35.433 [INFO] Kube API Test - Client Injecting Support ........... SKIPPED
17:24:35.433 [INFO] ------------------------------------------------------------------------
17:24:35.433 [INFO] BUILD FAILURE
17:24:35.433 [INFO] ------------------------------------------------------------------------
17:24:35.433 [INFO] Total time:  08:01 min
17:24:35.433 [INFO] Finished at: 2024-07-19T17:24:35+02:00
17:24:35.433 [INFO] ------------------------------------------------------------------------
17:24:35.434 [ERROR] Failed to execute goal on project io.fabric8.java-generator.gradle.plugin: Could not resolve dependencies for project io.fabric8.java-generator:io.fabric8.java-generator.gradle.plugin:jar:7.0-SNAPSHOT
17:24:35.434 [ERROR] dependency: org.gradle:gradle-tooling-api:jar:8.5 (provided)
17:24:35.434 [ERROR] 	org.gradle:gradle-tooling-api:jar:8.5 was not found in https://repo1.maven.org/maven2/ during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of maven-central has elapsed or updates are forced
17:24:35.434 [ERROR] 	org.gradle:gradle-tooling-api:jar:8.5 was not found in https://repo.gradle.org/gradle/libs-releases-local/ during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of repo.gradle.org has elapsed or updates are forced
17:24:35.434 [ERROR] 	org.gradle:gradle-tooling-api:jar:8.5 was not found in https://repo.gradle.org/gradle/ext-releases-local/ during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of ext.repo.gradle.org has elapsed or updates are forced
17:24:35.434 [ERROR] 	org.gradle:gradle-tooling-api:jar:8.5 was not found in https://repo1.maven.org/maven2/ during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of sonatype-maven-org has elapsed or updates are forced
17:24:35.434 [ERROR] 	org.gradle:gradle-tooling-api:jar:8.5 was not found in https://maven-central-eu.storage-download.googleapis.com/maven2/ during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of google-maven-central has elapsed or updates are forced
```


### Fabric8 Kubernetes Client version

SNAPSHOT

### Steps to reproduce

mvn clean package -DskipTests

### Expected behavior

Build success

### Runtime

Kubernetes (vanilla)

### Kubernetes API Server version

1.25.3@latest

### Environment

Linux

### Fabric8 Kubernetes Client Logs

_No response_

### Additional context

_No response_

--- a/pom.xml
+++ b/pom.xml
@@ -160,7 +160,7 @@
     <maven.invoker.plugin.version>3.9.0</maven.invoker.plugin.version>
     <maven.plugin.plugin.version>3.15.1</maven.plugin.plugin.version>
     <gmavenplus-plugin.version>4.1.1</gmavenplus-plugin.version>
-    <gradle-api-maven-plugin.version>0.0.5</gradle-api-maven-plugin.version>
+    <gradle-api-maven-plugin.version>0.0.7</gradle-api-maven-plugin.version>
     <jacoco-maven-plugin.version>0.8.12</jacoco-maven-plugin.version>
     <jkube.version>1.18.0</jkube.version>
     <sonar-maven-plugin.version>4.0.0.4121</sonar-maven-plugin.version>

